File: java/src/java/com/d_project/qrcode/ErrorCorrectLevel.java
package com.d_project.qrcode;

/**
 * 誤り訂正レベル.
 * @author Kazuhiko Arase 
 */
public interface ErrorCorrectLevel {

	/**
	 * 復元能力 7%.
	 */
	int L = 1;
	
	/**
	 * 復元能力 15%.
	 */
	int M = 0;

	/**
	 * 復元能力 25%.
	 */
	int Q = 3;

	/**
	 * 復元能力 30%.
	 */
	int H = 2;
	
}

File: ts/src/ts/com/d_project/qrcode/ErrorCorrectLevel.ts
'use strict';
namespace com.d_project.qrcode {

  /**
   * ErrorCorrectLevel
   * @author Kazuhiko Arase
   */
  export enum ErrorCorrectLevel {

    /**
     * 7%
     */
    L = 1,

    /**
     * 15%
     */
    M = 0,

    /**
     * 25%
     */
    Q = 3,

    /**
     * 30%
     */
    H = 2
  }
}
